bin/whisper-fill.py

```diff
@@ -118,20 +119,27 @@ def fill_archives(src, dst, startFrom):
         startFrom = fromTime
 
 
-def main(argv):
-        if len(argv) != 2:
-                print("usage: whisper-fill.py src dst");
-                print("       copies data from src in dst, if missing")
-                if len(argv) == 1 and (argv[0].lower() == "--help" or argv[0].lower() == "-h"):
-                        sys.exit(0)
+def main():
+        option_parser = optparse.OptionParser(
```

Comment:
I'm not sure this matters, but optparse is python 2.3-2.7 only AFAIK. Not that we are targeting py3k support anyways.